src/styles/blocks/contact-us.scss

```diff
@@ -0,0 +1,132 @@
+.contact-us {
+  &__block {
+    @include page-grid;
+
+    &-form {
+      grid-column: span 2;
+
+      @include on-tablet {
+        grid-column: span 6;
+      }
+    }
+  }
+
+  &__form {
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+
+    margin-bottom: 48px;
+  }
+
+  &__input {
+    height: 48px;
+    padding-left: 16px;
+
+    color: $main-text-color;
+
+    background-color: $field-color;
+    border: 1px solid $field-color;
+    border-radius: 8px;
+
+    &::placeholder {
+      font-size: 14px;
+      letter-spacing: 0.15px;
+      color: $inactive-text-color;
+    }
+
+    &:hover {
+      border-color: $hover-text-color;
+
+      &::placeholder {
+        color: $main-text-color;
+      }
+    }
+
+    &:focus {
+      outline-color: $accent-color;
+    }
+  }
+
+  &__textarea {
+    height: 148px;
+    padding: 14px 16px;
+    margin-bottom: 32px;
+    resize: none;
+
+    font-family: Manrope, sans-serif;
+    color: $main-text-color;
```

Comment:
u can create `normalize.scss` file for it. Move it. Use `html` tag for it

Reply:
I tried, but in this case i can't do this, because textarea can't inherit font-family.